[2025-07-23T09:22:54.942+0000] {scheduler_job_runner.py:781} ERROR - Executor reports task instance <TaskInstance: bitcoin_price_dag.fetch_bitcoin_price manual__2025-07-23T09:12:50.560481+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
